data/undergrad.yaml

```diff
@@ -294,14 +294,14 @@ Single Variable Real Analysis:
     connected subsets of $\R$: 'set_of_is_preconnected_eq_of_ordered'
     additive subgroups of $\R$:
   Numerical Series:
-    Convergence of real valued-series:
+    Convergence of real valued-series: 'has_sum'
     Geometric series: 'has_sum_geometric_of_abs_lt_1'
-    Riemann series:
-    positive valued series:
-    summation of comparison relations:
+    Riemann series (i.e., individual values of the Riemann zeta function):
+#    positive valued series: (this is certainly a concept that can be expressed using the definitions in mathlib, but there is no particular theorem)
```

Comment:
No, we cannot express this yet since we don't even have series. This item is a bit vague but it refers to a lot of results that hold only for series with positive values (for instance if such a series has general term bounded above by the term of a convergent series then it converges).

Reply:
We have `summable_of_norm_bounded` for the theorem you mentioned, but I guess it's a question of how literally to take the list.  Anyway, just let me know what you'd like to do -- should I clear the whole series section or only the ones you marked?  Feel free to make the changes yourself, also.

Reply:
I mean clearing the whole series section. It will be easy to fill once someones starts putting the relevant definitions and API, but we have none of that yet. This is not so important. I guess this whole section is very French, and for a very stupid reason: convergence of numerical series is great to create tricky technical exercises, and France has a lot of competitive exams at that level.

Reply:
Ok, I cleared it all, including the pre-existing entry for geometric series (but I can put that one back if you prefer).